File: app/src/main/java/com/packg/easynotes/Elements/CheckBox.java
package com.packg.easynotes.Elements;

public class CheckBox {
    protected boolean checked;
    protected String text;
    protected String type;

    public CheckBox(boolean checked, String text){
        this.checked = checked;
        this.text = text;
        this.type = "CheckBox";
    }

    public boolean isChecked() {
        return checked;
    }

    public void setChecked(boolean checked) {
        this.checked = checked;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }
}

File: app/src/main/java/com/packg/easynotes/Elements/CheckBox.kt
package com.packg.easynotes.Elements

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "check_box_table")
class CheckBox (@ColumnInfo(name = "name") val name : String,
                @ColumnInfo(name = "checked") val checked : Int){

    @ColumnInfo(name = "id")
    @PrimaryKey(autoGenerate = true) var id: Long = 0
}